SDWebImage/UIView+WebCache.m

```diff
@@ -52,31 +52,57 @@ - (void)sd_internalSetImageWithURL:(nullable NSURL *)url
         
         __weak __typeof(self)wself = self;
         id <SDWebImageOperation> operation = [SDWebImageManager.sharedManager loadImageWithURL:url options:options progress:progressBlock completed:^(UIImage *image, NSData *data, NSError *error, SDImageCacheType cacheType, BOOL finished, NSURL *imageURL) {
+            // in fact, this completed block is **always** called from main queue. Maybe we should do some cleanup
```

Comment:
we should not change this, we cannot be sure this block is always called on the main queue. 
That is the purpose of `dispatch_main_async_safe`. Will just check.

Reply:
OK. Keep these here.